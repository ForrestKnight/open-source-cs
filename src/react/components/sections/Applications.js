/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Applications extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Applications" className="Applications">
                
            </div>
        );
    }
}


export default Applications;