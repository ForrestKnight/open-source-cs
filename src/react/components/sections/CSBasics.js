/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class CSBasics extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="CSBasics" className="CSBasics">
                
            </div>
        );
    }
}


export default CSBasics;