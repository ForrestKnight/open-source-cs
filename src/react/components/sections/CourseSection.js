/* Packages */
import React from 'react';
import Course from '../Course';


/* Component */
export class CourseSection extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id={this.props.name} className="CourseSection">
                
            </div>
        );
    }
}


export default Math;