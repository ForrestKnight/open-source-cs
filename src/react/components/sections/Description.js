/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Description extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Description" className="Description">
                
            </div>
        );
    }
}


export default Description;