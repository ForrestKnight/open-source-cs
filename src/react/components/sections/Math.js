/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Math extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Math" className="Math">
                
            </div>
        );
    }
}


export default Math;