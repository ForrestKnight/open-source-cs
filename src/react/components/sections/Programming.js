/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Programming extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Programming" className="Programming">
                
            </div>
        );
    }
}


export default Programming;