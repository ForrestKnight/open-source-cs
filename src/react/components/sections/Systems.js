/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Systems extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Systems" className="Systems">
                
            </div>
        );
    }
}


export default Systems;