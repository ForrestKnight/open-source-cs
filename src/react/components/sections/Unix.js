/* Packages */
import React from 'react';
import {connect} from 'react-redux';


/* Component */
export class Unix extends React.Component {
    constructor(props) {
        super(props);
    }


    render() {
        return (
            <div id="Unix" className="Unix">
                
            </div>
        );
    }
}


export default Unix;