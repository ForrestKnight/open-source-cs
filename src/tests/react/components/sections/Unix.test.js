/* Packages */
import React from 'react';
import {shallow} from 'enzyme';


/* Component */
import {Unix} from '../../../../react/components/sections/Unix';


/* Test Config */
let component;
beforeEach(() => {
    component = shallow(<Unix />);
});


/* Tests */
test('Render Unix section', () => {
    expect(component).toMatchSnapshot();
});

