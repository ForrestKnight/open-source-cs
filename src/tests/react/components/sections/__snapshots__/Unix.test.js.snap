// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Unix section 1`] = `
<div
  className="Unix"
  id="Unix"
/>
`;
